kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from xmlrpc.client import Fault
+import pprint
+
+from kobo.client import ClientCommand
+
+
+class Create_Worker(ClientCommand):
+    """create a worker"""
+    enabled = True
+    admin = True
+
+
+    def options(self):
+        self.parser.usage = "%%prog %s worker_name [worker_name]" % self.normalized_name
+
+    def run(self, *args, **kwargs):
+        if not args:
+            self.parser.error("No worker name specified.")
+
+        username = kwargs.pop("username", None)
+        password = kwargs.pop("password", None)
+        hub = kwargs.pop("hub", None)
+        workers = args
+
+        self.set_hub(username, password, hub)
+        for worker in workers:
+            try:
+                new_worker = self.hub.client.create_worker(worker)
+                pprint.pprint(new_worker)
```

Comment:
@lzaoral Is it a good idea to pretty print if the output of the client is going to be processed by a script? That is, we will need to extract worker name and key from the output.

Reply:
That's true...  What about using `json` module instead given that the returned object can be serialised easily?

Reply:
I have modified the code to use `json` module.